### PR TITLE
Adding retry port TCP option to Servlet 4.0 Fat_2

### LINKED ISSUE #30349: Retry Option for Port Binding in Servlet 4.0 fat_2
The `<tcpOption>` for default configurations configures the `portOpenRetries` options to a value of `60` . However, since these tests configure non default `<httpEndpoint>` elements, the retry option is set to the Liberty TCP default of `0`. 

See the `<tcpOption>` configuration values [here](https://www.ibm.com/docs/en/was-liberty/nd?topic=configuration-tcpoptions). 

This bucket should add the retry option in order to avoid intermittent binding issues as the endpoints are shutdown and restarted quickly between tests .

### PR DESCRIPTION
**Pull request by mrsaldana**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Fixes #30349

### CODE REVIEW

**Comment by mrsaldana** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=ab1e89d5-71ff-464b-9bab-dab763b45ce2) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

**Review by isaacrivriv — APPROVED** [review]
Changes look good to me. Thanks for working on this!

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_headers/server.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2021 IBM Corporation and others.
+    Copyright (c) 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_headers/server.xml:2]: This should be a date range 2021, 2024

Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_headers/server.xml:2]: fixing this here: https://github.com/OpenLiberty/open-liberty/pull/30358

**Pull request merged**